src/main/java/org/cloudfoundry/autosleep/worker/remote/CloudFoundryApi.java

```diff
@@ -129,15 +132,14 @@ public void bindServiceInstance(List<ApplicationIdentity> applications, String s
     }
 
     @Override
-    public void bindServiceToRoute(String serviceInstanceId, String routeId, Map<String, Object> params)
-            throws CloudFoundryException {
+    public void bindServiceToRoute(String serviceInstanceId, String routeId) throws CloudFoundryException {
         log.debug("bindRouteToService");
         try {
             cfClient.serviceInstances()
                     .bindToRoute(BindServiceInstanceToRouteRequest.builder()
                             .serviceInstanceId(serviceInstanceId)
                             .routeId(routeId)
-                            .parameters(params).build())
+                            .parameters(null).build())
```

Comment:
you are right. Unecessary call due to bad refactoring. won't happen again :smile: